### PR TITLE
Add Fusion licensing note

### PR DESCRIPTION
**Pull request by justinegeffen**
Note for licensing.

### CODE REVIEW

**Comment by drewdipalma** [comment]
Can we add a note at the top-level (/fusion) as well?

File: fusion_docs/guide.mdx [hunk 1]
```diff
@@ -6,14 +6,15 @@ title: User guide
 
 Fusion is a virtual, lightweight, distributed file system designed to optimise the data access of Nextflow data pipelines.
 
-Fusion enables seamless filesystem I/O to cloud object stores via a standard POSIX interface resulting in simpler
-pipeline logic and faster, more efficient pipeline execution.
+Fusion enables seamless filesystem I/O to cloud object stores via a standard POSIX interface resulting in simpler pipeline logic and faster, more efficient pipeline execution.
+
+:::note
+Fusion is licensed by Seqera and can be used in Seqera Platform or integrated directly with Nextflow.
```
Comment (drewdipalma) [inline fusion_docs/guide.mdx:12]: How about: "Fusion requires a license, provided by Seqera, which enables usage within Seqera Platform or directly with Nextflow."

Comment (pditommaso) [reply fusion_docs/guide.mdx:12]: Is the plan having a free usage quota that's essentially unlicensed? I think this wording will keep potential users even from evaluate it.

Comment (drewdipalma) [reply fusion_docs/guide.mdx:12]: Good point, we can incorporate that into something like – 

"Fusion permits trial/evaluation usage for free but production usage requires a license.  Licenses are provided by Seqera and enable usage within Seqera Platform or directly with Nextflow."

Gets a bit wordy but I think it gets the points across.

Comment (justinegeffen) [inline fusion_docs/guide.mdx:12]:
```suggestion
You can try out Fusion for free. To use Fusion in a production environment, a license is required. Fusion is licensed by Seqera for use in Seqera Platform or directly with Nextflow."
```

Comment (llewellyn-sl) [reply fusion_docs/guide.mdx:12]: @justinegeffen Production use is a pretty nebulous term here that may cause confusion. Do we know what the exact use limits for the trial will be? Something like "Fusion is free to evaluate for X (time/pipeline runs/GB data/etc.). Thereafter, a license is required to use Fusion in Seqera Platform or Nextflow."

Comment (justinegeffen) [reply fusion_docs/guide.mdx:12]: Great points, @llewellyn-sl. Thank you. I'm committing my change so it's easier to see where we're at, but will revert on this ASAP.

File: fusion_docs/guide.mdx [hunk 2]
```diff
@@ -6,14 +6,15 @@ title: User guide
 
 Fusion is a virtual, lightweight, distributed file system designed to optimise the data access of Nextflow data pipelines.
 
-Fusion enables seamless filesystem I/O to cloud object stores via a standard POSIX interface resulting in simpler
-pipeline logic and faster, more efficient pipeline execution.
+Fusion enables seamless filesystem I/O to cloud object stores via a standard POSIX interface resulting in simpler pipeline logic and faster, more efficient pipeline execution.
+
+:::note
+Fusion is free for trial purposes. To use Fusion in a production environment, a license is required. Fusion is licensed by Seqera for use in Seqera Platform or directly with Nextflow.
```
Comment (llewellyn-sl) [inline fusion_docs/guide.mdx:12]: Same note as before. Fusion is free to evaluate, but what constitutes production use is nebulous, and largely irrelevant. If we can state the exact terms of the evaluation period, we can simplify this.

Comment (llewellyn-sl) [reply fusion_docs/guide.mdx:12]: "Fusion is free to evaluate for X (time/pipeline runs/GB data/etc.). Contact us for a license to use Fusion in Seqera Platform or Nextflow."

Comment (rhondasilva) [reply fusion_docs/guide.mdx:12]: Very much agree with Llewellyn. I don't want people to think "production use" means running production pipelines - if they are using Fusion to iteratively develop pipelines, that is still non-trial / use of Fusion. 

The limits of the free usage have now been defined (though not technically implemented) - should we note them here now or is it too premature? Going to talk to drew about this in ~30

Comment (rhondasilva) [reply fusion_docs/guide.mdx:12]: OR! something like this 

Fusion requires a license for use within Seqera Platform or with Nextflow outside of Seqera Platform. Try Fusion for free by following the steps below:

Comment (drewdipalma) [reply fusion_docs/guide.mdx:12]: I'd like to avoid the specific numbers until they're implemented, but this slight adjustment on what you proposed works for me – "Fusion requires a license for use beyond limited testing and validation within Seqera Platform or directly within Nextflow. Contact Seqera for more details."

Comment (justinegeffen) [inline fusion_docs/guide.mdx:12]:
```suggestion
Fusion requires a license for use beyond limited testing and validation within Seqera Platform or directly within Nextflow. Contact Seqera for more details.
```

File: fusion_docs/index.mdx [hunk 3]
```diff
@@ -11,6 +11,10 @@ Fusion is a virtual, lightweight, distributed file system that bridges the gap b
 storage. Fusion enables seamless filesystem I/O to cloud object stores via a standard POSIX interface resulting in
 simpler pipeline logic and faster, more efficient pipeline execution.
 
+:::note 
+Fusion is licensed by Seqera and can be used in Seqera Platform or integrated directly with Nextflow.
```
Comment (justinegeffen) [inline fusion_docs/index.mdx:15]:
```suggestion
Fusion requires a license for use beyond limited testing and validation within Seqera Platform or directly within Nextflow. Contact Seqera for more details.
```

**Review by drewdipalma — APPROVED** [review]

File: fusion_docs/guide.mdx [hunk 4]
```diff
@@ -6,14 +6,15 @@ title: User guide
...
+:::note
+Fusion requires a license for use beyond limited testing and validation within Seqera Platform or directly within Nextflow. Contact Seqera for more details.
+:::
 
 ## Getting started
 
-Fusion smoothly integrates with Nextflow and does not require any installation or change in pipeline code.
-It only requires to use of container runtime or a container computing service such as Kubernetes, AWS Batch and
-Google Cloud Batch.
+Fusion smoothly integrates with Nextflow and does not require any installation or change in pipeline code. It only requires to use of container runtime or a container computing service such as Kubernetes, AWS Batch and Google Cloud Batch.
```
Comment (llewellyn-sl) [inline fusion_docs/guide.mdx:17]:
```suggestion
Fusion smoothly integrates with Nextflow and does not require any installation or change in pipeline code. It only requires to use of container runtime or a container computing service such as Kubernetes, AWS Batch, or Google Cloud Batch.
```

File: fusion_docs/index.mdx [hunk 5]
```diff
@@ -11,6 +11,10 @@ Fusion is a virtual, lightweight, distributed file system that bridges the gap b
 storage. Fusion enables seamless filesystem I/O to cloud object stores via a standard POSIX interface resulting in
 simpler pipeline logic and faster, more efficient pipeline execution.
 
+:::note 
+Fusion requires a license for use beyond limited testing and validation within Seqera Platform or directly within Nextflow. Contact Seqera for more details.
```
Comment (llewellyn-sl) [inline fusion_docs/index.mdx:15]: Maybe a link to sales here.

**Review by llewellyn-sl — APPROVED** [review]
Thanks! Approving, contingent  on those small points.

File: fusion_docs/guide.mdx [hunk 6]
```diff
@@ -6,14 +6,15 @@ title: User guide
 
 Fusion is a virtual, lightweight, distributed file system designed to optimise the data access of Nextflow data pipelines.
 
-Fusion enables seamless filesystem I/O to cloud object stores via a standard POSIX interface resulting in simpler
-pipeline logic and faster, more efficient pipeline execution.
+Fusion enables seamless filesystem I/O to cloud object stores via a standard POSIX interface resulting in simpler pipeline logic and faster, more efficient pipeline execution.
+
+:::note
+Fusion requires a license for use beyond limited testing and validation within Seqera Platform or directly within Nextflow. Contact Seqera for more details.
```
Comment (justinegeffen) [inline fusion_docs/guide.mdx:12]:
```suggestion
Fusion requires a license for use beyond limited testing and validation within Seqera Platform or directly within Nextflow. [Contact Seqera](https://seqera.io/contact-us/) for more details.
```

File: fusion_docs/index.mdx (hunk 5 above)
Comment (justinegeffen) [inline fusion_docs/index.mdx:15]:
```suggestion
Fusion requires a license for use beyond limited testing and validation within Seqera Platform or directly within Nextflow. [Contact Seqera](https://seqera.io/contact-us/) for more details.
```
